src/Scrollable/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import useIsVisible from '../hooks/useIsVisible';
+
+export const CLASSNAME_SCROLL_TOP = 'pgn__scrollable-body-scroll-top';
+export const CLASSNAME_SCROLL_BOTTOM = 'pgn__scrollable-body-scroll-bottom';
+
+const Scrollable = ({ children, ...props }) => {
+  const [isScrolledToTop, topSentinelRef] = useIsVisible();
+  const [isScrolledToBottom, bottomSentinelRef] = useIsVisible();
+  const className = classNames(
+    'pgn__scrollable-body',
+    props.className,
+    {
+      [CLASSNAME_SCROLL_TOP]: isScrolledToTop,
+      [CLASSNAME_SCROLL_BOTTOM]: isScrolledToBottom,
+    },
+  );
+  return React.createElement(
+    'div',
+    { ...props, className },
+    (
```

Comment:
If you don't intend to offer an `as` prop to allow control of the element type. You might consider going back to jsx syntax: 

```
<div {...props} className={className}>
 ...
```

Reply:
yeah I like this thanks